VK_App/NetworkManager/Operation/ReloadTableController.swift

```diff
@@ -0,0 +1,24 @@
+// ReloadTableController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import RealmSwift
+
+/// Обновление данных в реалм
+final class ReloadTableController: Operation {
```

Comment:
```suggestion
final class ReloadTable: Operation {
```
Controller это не контролер